templates/default/ckan_properties.ini.erb

```diff
@@ -177,8 +177,8 @@ ckan.resource_proxy.chunk_size = 8192
 # package_hide_extras = for_search_index_only
 #package_edit_return_url = http://another.frontend/dataset/<NAME>
 #package_new_return_url = http://another.frontend/dataset/<NAME>
-#ckan.recaptcha.publickey =
-#ckan.recaptcha.privatekey =
+ckan.recaptcha.publickey = ${ssm:/config/CKAN/<%= node['datashades']['version'] %>/app/<%= node['datashades']['app_id'] %>/recaptcha.publickey}
```

Comment:
I would probably use `recaptcha/publickey` instead of a dot, but okay.

Reply:
I've already loaded the data so would need to redo all environments to change over if pushed.